helm/exporter-kube-controller-manager/templates/kube-controller-manager.rules.yaml

```diff
@@ -3,7 +3,7 @@ groups:
 - name: kube-controller-manager.rules
   rules:
   - alert: K8SControllerManagerDown
-    expr: absent(up{job="{{ template "fullname" .  }}"} == 1)
+    expr: absent(up{job="{{ template "exporter-kube-controller-manager.fullname" .  }}"} == 1)
```

Comment:
All the helm/*/rules.yaml are generated by [sync_kube_prometheus](https://github.com/coreos/prometheus-operator/blob/master/helm/hack/sync_kube_prometheus.py) script. Rather than edit manually this files, I would recommend to make the changes on the script and generate all the rules.

Reply:
ok, will do. thank's for the info